content/03.prior-knowledge.cascadelearning.md

```diff
@@ -0,0 +1,30 @@
+## Cascade Learning {.page_break_before}
+
+Sometimes, due to paucity of data and its unstable predictive capabilities, one may feel the need to consult various machine learning methods and the predictions there-of to make an educated decision about the identified predictive patterns, collectively called _ensemble learning_. 
```

Comment:
I would probably rephrase `unstable predictive capabilities` – this strikes me as a bit of jargon.